core/server/services/stats/index.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Stats
+ * A collection of utilities for handling stats of the site
+ */
+const debug = require('ghost-ignition').debug('stats:cache');
+const moment = require('moment');
+const models = require('../../models');
+const StatsCache = require('./cache');
+
+async function initStats() {
+    const result = await models.Post.query(qb => qb.orderBy('created_at', 'ASC').limit(1)).fetchAll({columns: ['created_at']});
+    const createdAt = result.models[0].get('created_at');
```

Comment:
Right. But the problem is, when should we set the `createdAt`? If we set it to `null`, it can break the theme.

I think there are options:

* current time: it's simple and fast (no more query). But in this case, the name and the value doesn't go well together.
* the oldest page time: if the blog has no page again, we encounter the same problem. 
* the timestamp of `db_hash` value at `settings` table: I think it's the best alternative because it's created when we set up a Ghost blog and it is unlikely that it would be deleted by the system. (The only possibility is the malicious attacks or the mistake by the user.) 

I think `db_hash` is the best fallback for the created date of the first post. What do you think?

Reply:
Have asked a question in the main issue - https://github.com/TryGhost/Ghost/issues/12031#issuecomment-703490561. Let's see what the outcome is.

Reply:
@sainthkh fyi the question above has been answered [here](https://github.com/TryGhost/Ghost/issues/12031#issuecomment-704972385). Hopefully that unblocks next steps in the implementation ;)